ui/admin/app/components/form/worker/create-worker-led/index.js

```diff
@@ -149,6 +149,21 @@ boundary server -config="${this.configFilePath || '<path>'}/pki-worker.hcl"`;
       .join('\n    ');
   }
 
+  getClusterIdFromURL() {
+    const hostname = this.browserObject.window.location.hostname;
```

Comment:
Perhaps the location/hostname should also be part of the browser service, if not this entire method.

Reply:
I can make the hostname as another getter in the service

Reply:
Conversely, if hostname is provided as a getter here, then you only have to stub/mock `service.hostname` in your tests.

Reply:
Sure, it's not much different either way so I included it